test/main_contracts/rate_oracles/compoundBorrowConfig.ts

```diff
@@ -0,0 +1,44 @@
+import { MockCToken, TestRateOracle } from "../../../typechain";
+import { toBn } from "../../helpers/toBn";
+import { deployments, ethers } from "hardhat";
+
+let cToken: MockCToken;
+const startingExchangeRate = 1;
+
+export const ConfigForGenericTests = {
+  configName: "CompoundBorrow",
+  startingExchangeRate,
+  oracleFixture: async () => {
+    // Use hardhat-deploy to deploy factory and mocks
+    await deployments.fixture(["Factory", "Mocks"]);
+    const token = await ethers.getContract("ERC20Mock");
+
+    // store the cToken and interest rate model and some rate per block info for use when setting rates
+    cToken = (await ethers.getContract("MockCToken")) as MockCToken;
+
+    // Set inital rate per block to 0
+    await cToken.setBorrowRatePerBlock(0);
+
+    // set initial borrow index
+    await cToken.setBorrowIndex(toBn(startingExchangeRate, 18));
+
+    // set last update block
+    const latestBlock = await ethers.provider.getBlock("latest");
+    await cToken.setAccrualBlockNumber(latestBlock.number);
```

Comment:
this setup means there is no interpolation, are we able to test the interpolation logic works as expected?